modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less

```diff
@@ -1,21 +1,22 @@
 //
 //
 // Accessibility checker
+@overlay-opacity: 80%;
```

Comment:
Shouldn't this be named `@accessibility-overlay-opacity` or something if that is specific to the a11ychecker?

Reply:
yeah true, fixed